services/user.py

```diff
@@ -0,0 +1,51 @@
+from django.contrib.auth import get_user_model
+from typing import Optional
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: Optional[str] = None,
+        first_name: Optional[str] = None,
+        last_name: Optional[str] = None
+) -> None:
+    user_create = User.objects.create_user(
+        username=username,
+        password=password,
+    )
+    if email:
+        user_create.email = email
+    if first_name:
+        user_create.first_name = first_name
+    if last_name:
+        user_create.last_name = last_name
```

Comment:
Move this code to separate function and remove duplicate of code from update_user